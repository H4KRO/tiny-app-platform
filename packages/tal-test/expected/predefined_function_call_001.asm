main()
  entry:
    Literal           0
    MakeArray
    Literal           0
    MakeRecord
    Local             give_hello
    Call
