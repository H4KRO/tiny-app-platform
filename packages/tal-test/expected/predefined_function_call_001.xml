<program>
  <call>
    <local />
  </call>
</program>
